src/this.js

```diff
@@ -4,8 +4,16 @@
 
 class User {
   constructor(options) {
+    this.username = options.username;
+    this.password = options.password;
     // set a username and password property on the user object that is created
   }
+  checkPassword(passwordToCompare) {
+    if (this.password === passwordToCompare) {
```

Comment:
This can be simplified to `return this.password === passwordToCompare;`
The `===` means that the expression will be evaluated into `true` or `false` before it is returned